### PR TITLE
i18n: accept dot notation for objects in strings.json 

### PR DESCRIPTION
**Pull request by raphink**
This contains patches for #848 and #849 because it depends on them.

### CODE REVIEW

File: public/js/simpleStrings-0.0.1.js [hunk 1]
```diff
@@ -34,10 +34,13 @@
         var text = item.clone().children().remove().end().text();
 
         if(matches = text.match(/{{([^}]*)}}/) ) {
+          matches.shift();
           $.each(matches, function() {
               var keyword = this;
-              if(keyword in settings.strings) {
-                  text = text.replace('{{'+keyword+'}}', settings.strings[keyword]);
+              var newText = keyword.split('.')
+                    .reduce((o, i) => o[i], settings.strings);
```
Comment (binford2k) [inline public/js/simpleStrings-0.0.1.js]: I'm going to guess that this eliminates IE support. Can we use the `.reduce(index, obj)` form instead?

**Review by binford2k — CHANGES_REQUESTED** [review]

File: public/js/simpleStrings-0.0.1.js (hunk 1 above)
Comment (raphink) [reply public/js/simpleStrings-0.0.1.js]: There, I fixed it.

**Comment by binford2k** [comment]
I've reworked this to apply on top of #852; when that's merged, I'll PR this work.

**Comment by binford2k** [comment]
Superseded by #854

**Pull request closed without merging**